requirements/core.txt

```diff
@@ -9,5 +9,5 @@ netcdf4<1.4
 numpy
 scipy
 # pyke (not pip installable)  #conda: pyke
-cf_units>=1.2
```

Comment:
It may be that I didn't need to do this at all...
I've asked a question in #3016 of what the driver for this is, and whether we would actually function with an older cf_units...